Source/Scene/Cesium3DTileset.js

```diff
@@ -903,6 +904,7 @@ function Cesium3DTileset(options) {
     .then(function (url) {
       var basePath;
       resource = Resource.createIfNeeded(url);
+      that._resource = resource;
```

Comment:
This actually seems like an oversight/bug to me. It's even odder to me that `options.url` can be a Resource object but ends up as a string on `tileset.url`.  Either way, it would probably  be better to add a public getter here for `tileset.resource` and deprecate `tileset.url` (it doesn't appear to actually be used anywhere).